synapse/cortex.py

```diff
@@ -2228,7 +2228,10 @@ async def _initCoreViews(self):
             assert self.inaugural, 'Cortex initialization failed: there are no views.'
             ldef = await self.addLayer(nexs=False)
             layriden = ldef.get('iden')
+            layr = self.getLayer(layriden)
+            await layr._hndlsetLayerInfo('name', 'default')
```

Comment:
this should probably just be provided in the ldef to addLayer() above rather than slicing it in ex-post-facto

Reply:
It isn't part of the ldef.

Reply:
We I just fixed it.